dom0/securedrop-handle-upgrade

```diff
@@ -47,6 +47,33 @@ if [[ $TASK == "prepare" ]]; then
 		fi
 	fi
 
+    # For Whonix VMs, shut them down, so we can upate the TemplateVM settings.
+    # We shut down sd-proxy before sd-whonix, since its netvm is sd-whonix, which won't
+    # shutdown if a client is connected.
+	if qvm-check --quiet sd-proxy; then
+		BASE_TEMPLATE=$(qvm-prefs sd-proxy template)
+		if [[ ! $BASE_TEMPLATE =~ "buster" ]]; then
+			qvm-shutdown --wait sd-proxy
+		fi
+	fi
+
+	if qvm-check --quiet sd-whonix; then
+		BASE_TEMPLATE=$(qvm-prefs sd-whonix template)
+		if [[ ! $BASE_TEMPLATE =~ "15" ]]; then
+			qvm-shutdown --wait sd-whonix
+		fi
+	fi
+
+    # Kill sys-whonix, to make sure connected clients don't prevent shutdown.
+	if qvm-check --quiet sys-whonix; then
+		BASE_TEMPLATE=$(qvm-prefs sys-whonix template)
+		if [[ ! $BASE_TEMPLATE =~ "15" ]]; then
+            if qvm-check --quiet --running sys-whonix; then
```

Comment:
@emkll Ack, this is tabs-versus-spaces. Most of the scripts in the repo use 4 spaces for bash scripts, but this one and only this one uses tabs instead—I wasn't careful about making sure to force use of tabs when editing the script. Agreed, we should indeed clean it up to avoid large amounts of frustration. :smiley: